about.html

```diff
@@ -197,7 +197,66 @@
               <h1 style="font-weight: 300">Introducing Aqovia</h1>
               <p style="color: #FFFFFF; max-width: 90%; width: 800px"> Digital solutions that empower businesses and human talent and offer new pathways to harness change - that is what we are all about. Our business centric value approach ensures low risk and high reward. </p>
           <p style="color: #FFFFFF">

```

Comment:
I know @ma499 asked me to focus on what have been added, but I feel I should really point out that having inline styles is really not a good idea.

Reply:
Thanks,
moved the code to style.css